lms/static/scripts/frontend_apps/index.js

```diff
@@ -60,6 +61,9 @@ switch (config.mode) {
   case 'oauth2-redirect-error':
     app = <OAuth2RedirectErrorApp />;
     break;
+  case 'error-dialog':
```

Comment:
Can you move this `case` statement above the previous one, so they are in sorted order by `mode`.